src/App.js

```diff
@@ -14,18 +15,26 @@ function App() {
     name: "Guest Account",
     avatar_url: "https://cdn-icons-png.flaticon.com/512/1077/1077114.png?w=360",
   });
+  const [selectedCategory, setSelectedCategory] = useState(
+    window.location.pathname.substring(1) || "all"
+  );
 
   return (
     <BrowserRouter>
       <UserContext.Provider value={user}>
-        <div className="app"></div>
-        <header>
-          <Header />
-          <NavBar />
-        </header>
-        <Routes>
-          <Route path="/" element={<ReviewList />} />
-        </Routes>
+        <CategoryContext.Provider
+          value={{ selectedCategory, setSelectedCategory }}
+        >
+          <div className="app"></div>
+          <header>
+            <Header />
+            <NavBar />
+          </header>
+          <Routes>
+            <Route path="/" element={<ReviewList />} />
+            <Route path="/:category" element={<ReviewList />} />
```

Comment:
Here you have a parametric endpoint with `category` as the parameter. You can easily access this with a useParams hook :)